### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by shrikantbarve** [ging/licode#462]
Hi,
I'm unable to install on Ubuntu 14. After some time, it keeps on giving some sort of syntax issues.
I believe there is a syntax problem in dependency tools. Please try to install latest version on fresh copy of Ubuntu 14.

Thanks

**Comment by cracker0dks** [comment]
yeah you get some warnings. But thats no problem. Installed yesterday and everything is working.

**Comment by shrikantbarve** [comment]
![err1](https://cloud.githubusercontent.com/assets/19504442/15450453/88569930-1fb9-11e6-9334-ceb200aded12.PNG)
![err2](https://cloud.githubusercontent.com/assets/19504442/15450454/887eb672-1fb9-11e6-8cd7-2e70d211385d.PNG)
![err3](https://cloud.githubusercontent.com/assets/19504442/15450455/887ef31c-1fb9-11e6-8953-a00d74cc92c6.PNG)
![err4](https://cloud.githubusercontent.com/assets/19504442/15450456/88852bce-1fb9-11e6-8f5c-710556e17921.PNG)
![err5](https://cloud.githubusercontent.com/assets/19504442/15450457/888577fa-1fb9-11e6-8827-71ec2b8b9619.PNG)
![err6](https://cloud.githubusercontent.com/assets/19504442/15450458/888e186a-1fb9-11e6-9313-f9ee3afdf0a0.PNG)

**Comment by cracker0dks** [comment]
Ok that are some errors :D 
was this a clean ubuntu14.04 system? 
my installation took ~35 mins

I recommend to use docker or vagrant so you can build easy on a clean state and swap your system to a new code base if you want to.

**Comment by nachoaguadoc** [comment]
Are the errors still taking place in your installation? Latest Licode version can be installed fine in Ubuntu 14.04, so it may be a problem of some dependencies you've installed before.

**Comment by shrikantbarve** [comment]
@nachoaguadoc Let me try again

**Comment by shrikantbarve** [comment]
@nachoaguadoc Still no luck.
![err1](https://cloud.githubusercontent.com/assets/19504442/15552568/9aa89184-22d8-11e6-8ebb-a5c6eb9673ad.PNG)
![err2](https://cloud.githubusercontent.com/assets/19504442/15552575/a2d3313e-22d8-11e6-8def-7f9c9372d229.PNG)

I will go ahead with Docker. I found https://hub.docker.com/r/lynckia/licode/
However this docker method works on which version of ubuntu ?
12.04 or 14.04 ?

**Comment by cracker0dks** [comment]
ubuntu:14.04 you can see this at the dockerfile. But you have to change the dockerfile and build on your own look at this: https://github.com/ging/licode/issues/453

**Comment by sonnh273** [comment]
Hello,

I got a lot of warning when tried to compile libav:
![libav_compile](https://cloud.githubusercontent.com/assets/374312/15848691/a39ded9a-2cb8-11e6-85d9-9bd2b65fd87a.png)
And finally, only video works fine, audio works only in p2p mode, not in MCU mode.
I installed on a clean Ubuntu 14.04

Any suggestions?

**Comment by cracker0dks** [comment]
syntax warnings at installation are not a problem (at least for me). If Video works audio should as well. Do you have any client/servers console errors?

**Comment by sonnh273** [comment]
@cracker0dks I think so, too. If the video works, the audio should work, too. That's why I think there may be a problem with the libav.
There is no error on both client side & server side, everything seems ok.
Here is the log on server after I published the audio stream (still no sound):
`
[2016-06-07 21:30:37.073] [INFO] ErizoJSController - Process Signaling message:  340044253971427650 2U2lnfzMlIsaEPQORWXU { type: 'candidate',
  candidate:
   { sdpMLineIndex: 0,
     sdpMid: 'audio',
     candidate: 'a=candidate:3592574086 1 udp 2122255103 2001::9d38:6ab8:34c5:9a52:2103:dd38 55488 typ host generation 0 ufrag Cjxua4j6NCpTBhJm' } }
[2016-06-07 21:30:37.079] [INFO] ErizoJSController - Process Signaling message:  340044253971427650 2U2lnfzMlIsaEPQORWXU { type: 'candidate',
  candidate:
   { sdpMLineIndex: 0,
     sdpMid: 'audio',
     candidate: 'a=candidate:3467446226 1 udp 2122194687 192.168.1.109 55489 typ host generation 0 ufrag Cjxua4j6NCpTBhJm' } }
[2016-06-07 21:30:37.156] [INFO] ErizoJSController - Process Signaling message:  340044253971427650 2U2lnfzMlIsaEPQORWXU { type: 'candidate',
  candidate:
   { sdpMLineIndex: 0,
     sdpMid: 'audio',
     candidate: 'a=candidate:2560731254 1 tcp 1518275327 2001::9d38:6ab8:34c5:9a52:2103:dd38 9 typ host tcptype active generation 0 ufrag Cjxua4j6NCpTBhJm' } }
[2016-06-07 21:30:37.161] [INFO] ErizoJSController - Process Signaling message:  340044253971427650 2U2lnfzMlIsaEPQORWXU { type: 'candidate',
  candidate:
   { sdpMLineIndex: 0,
     sdpMid: 'audio',
     candidate: 'a=candidate:2150562594 1 tcp 1518214911 192.168.1.109 9 typ host tcptype active generation 0 ufrag Cjxua4j6NCpTBhJm' } }
[2016-06-07 21:30:37.339] [DEBUG] ErizoJSController - webrtc Addon status  104  id pub 340044253971427650 id_sub 2U2lnfzMlIsaEPQORWXU
[2016-06-07 21:30:37.341] [INFO] ErizoController - Subscriber from client 2U2lnfzMlIsaEPQORWXU to stream 340044253971427650 is now ready
[2016-06-07 21:30:37.340] [INFO] ErizoJSController - Connection id_pub: 340044253971427650 idsub: 2U2lnfzMlIsaEPQORWXU ready
`

**Comment by cracker0dks** [comment]
on witch browser are you testing ? Have you tested with the basicExample ? This app gives you a nice log on the gui about the media/stream status.

**Comment by shrikantbarve** [comment]
![docker_licode_error](https://cloud.githubusercontent.com/assets/19504442/16229363/fcaba3d8-37d9-11e6-87d4-5f8e41dd5559.PNG)

Docker is hanging up once I start it.

**Comment by cracker0dks** [comment]
yeah! witch docker image is this? Have you set the HOSTIP to the server ip?

**Comment by btamayo** [comment]
@shrikantbarve: The problem with your installation with openssl is the same as this one [here](https://github.com/shadow/shadow/issues/206) and can be fixed by modifying the line in `installUbuntuDepsUnattended.sh` or `installUbuntuDeps.sh` under `install_openssl()` from `make install` to `make install_sw` which installs openssl without its man pages, which seems to be formatted in a way that isn't compatible with Ubuntu 14.04
